en/docs/references/ballerina-expression-syntax.md

```diff
@@ -326,7 +325,7 @@ null
 
 Ballerina represents JSON objects as maps. You can define a JSON object to hold values of any `json` type (`string`, `number`, `boolean`, `object`, `array`, or `null`). You can declare a  JSON object in two ways:
     -  Using the variable type `json`.
-    -  Using the variable type `other` and `map<json>` as the `other type`.
+    -  Using the variable type `other` and `map<json>` as the `Other Type`.
```

Comment:
`Other` is not a type in ballerina. In choreo `Other` is used to indicate that user want to select a type which is not mentioned by the drop down. So if we word  it like this it is confusing, specically this doc is about ballerina syntax guide. We need to say why select other and then do what.

Reply:
Removed `Using the variable type `other` and `map<json>` as the `Other Type`.